examples/dq/README.md

```diff
@@ -55,3 +55,8 @@ Creates a constraint that asserts on a column's allowed values.\
 Required parameter: `column` \
 Required parameter: `allowedValues` (List of strings)
 
+### Failures
+To help debug dataframes that failed verification, they will be stored as Parquet files.
+To specify a datastore you can use the `CONFIG_FAILED_DF_PATH_PREFIX` environment variable.
```

Comment:
I thought we have a default..?

Reply:
We use the deployment templates to set that variable, so anyone using DQ in _production_ will get a default value. It is just not stored in this repo.

Reply:
I know, but the next comment says if I don't specify a path the DF will not be saved (which to my understanding is a blatant lie..)

Reply:
That's exactly what will happen. I did it to simplify running it locally, where you probably don't need that feature. And if you do - you can always add the env variable locally.